src/node/server.ml

```diff
@@ -29,6 +29,19 @@ let rec reset_timeout server =
      | Error `Not_current_block_producer -> ());
      reset_timeout server;
      Lwt.return_unit)
+
+let task_pool = ref None
```

Comment:
Not a fan, but I understand the why. I think the different parts of Deku's initialization should appear all together somewhere in the future (probably bin/node_state) to be more explicit. What do you think? @EduardoRFS @d4hines

Reply:
Agreed with Daniel. Almost all global state is initialized at server.ml, the remaining are state that are not relevant to us, such as the RNG.

Reply:
Agreed.